engo_js.go

```diff
@@ -191,29 +152,79 @@ func rafPolyfill() {
 		})
 	}
 
-	if window.Get("cancelAnimationFrame").IsUndefined() {
+	if window.Get("cancelAnimationFrame") == nil {
 		window.Set("cancelAnimationFrame", func(id int) {
 			js.Global.Get("clearTimeout").Invoke(id)
 		})
 	}
 }
 
-func RequestAnimationFrame(callback func(float32)) int {
+func RunIteration() {
+	keysUpdate()
+	currentWorld.Update(Time.Delta())
+	Time.Tick()
+	// TODO: this may not work, and sky-rocket the FPS
```

Comment:
I thought it would, but if it's anything like I wrote a while back; it runs at 60fps.